### ISSUE TITLE
Cannot install with redis-smq 7.0.1

### ISSUE THREAD

**Issue by PhilHannent** [weyoss/redis-smq-monitor#1]
There is a problem where I cannot install this library due to:

```
Found: redis-smq@7.0.1
npm ERR! node_modules/redis-smq
npm ERR!   redis-smq@"^7.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! node_modules/redis-smq-monitor
npm ERR!   redis-smq-monitor@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

```

**Comment by weyoss** [comment]
@PhilHannent Thank you for opening this issue.

Peer dependencies has been fixed in redis-smq-monitor@7.0.2

**Comment by weyoss** [comment]
Closing as resolved.
